apps/website/src/components/modules/Home/Sections/Donate.tsx

```diff
@@ -0,0 +1,27 @@
+// components/WhyDevLaunchers.js
+import React from 'react';
+import { Section } from './Section';
+import { styles } from '../styles';
+import { sections } from './sections';
+
+const DonateSection = () => {
+  return (
+    <Section className="">
+      <div className={styles.section.header}>
+        <h2 className={styles.section.heading}>{sections.donate.title}</h2>
+        <p className={styles.section.subHeading}>
+          {sections.donate.description}
+        </p>
+      </div>
+      <div className={styles.section.buttonContainer}>
+        <a href="https://devlaunchers.org/support-us">
```

Comment:
Should we get the hostname from `process.env.NEXT_PUBLIC_API_URL`, so we can link to the dev or staging version of the support page?

Reply:
I changed it to `/support-us`. It works both on dev and production environments.